src/components/BookForm.jsx

```diff
@@ -24,20 +29,24 @@ const BookForm = () => {
 
   return (
     <form onSubmit={handleSubmit}>
-      <h2>ADD NEW BOOK</h2>
-      <input
-        type="text"
-        placeholder="Title"
-        value={title}
-        onChange={(e) => setTitle(e.target.value)}
-      />
-      <input
-        type="text"
-        placeholder="Author"
-        value={author}
-        onChange={(e) => setAuthor(e.target.value)}
-      />
-      <button type="submit">ADD BOOK</button>
+      <fieldset>
+        <legend><h2>ADD NEW BOOK</h2></legend>
+        <input
+          className="title"
+          type="text"
+          placeholder="Title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <input
+          className="author"
+          type="text"
+          placeholder="Author"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+        />
+        <button className="submit-btn" type="submit">ADD BOOK</button>
+      </fieldset>
     </form>
```

Comment:
Your design
![image](https://github.com/coker2019/booksave/assets/78251927/1a21cfb8-4704-4956-94fc-aea04149b95e)

Template
![image](https://github.com/coker2019/booksave/assets/78251927/af18d54c-d0c5-4071-9d8e-738247c4f84a)

Kindly align the text `ADD NEW BOOK` to the left and adjust the ` font-size` to match the template 🤗🛠👍